es6-persistence/src/main/java/com/netflix/conductor/es6/config/ElasticSearchProperties.java

```diff
@@ -205,4 +209,29 @@ private URL toURL(String url) {
             throw new IllegalArgumentException(url + "can not be converted to java.net.URL");
         }
     }
+
+    public List<URI> getURIs() {
+
+        String clusterAddress = getURL();
+
+        String[] hosts = clusterAddress.split(",");
+
+        return Arrays.stream(hosts).map(host ->
+                (host.startsWith("http://") || host.startsWith("https://") || host.startsWith("tcp://")) ? URI.create(host) : URI.create("tcp://" + host)
+        ).collect(Collectors.toList());
+    }
+
+    private String getProperty(String key, String defaultValue) {
+        String val;
+        val = System.getenv(key.replace('.', '_'));
+        if (val == null || val.isEmpty()) {
+            val = Optional.ofNullable(System.getProperty(key))
+                    .orElse(defaultValue);
+        }
+        return val;
+    }
+
+    private String getURL() {
```

Comment:
Is there a reason this method is needed? Doesn't `getUrl()` do the same?

Reply:
Yes, hence removed newly introduced getURL() and used existing getUrl() of ElasticSearchProperties.java class